lib/scnnr/connection.rb

```diff
@@ -5,11 +5,16 @@ class Connection
     require 'net/http'
     require 'json'
 
-    def initialize(uri, method, api_key, logger)
+    MAX_SLEEP_ON_RETRY = 5
+    MAX_RETRY_LIMIT = 15
```

Comment:
If we fail to connect 3 times or more continuously, I suspect there is a serious network problem and retrying many times basically does not solve the problem.

Should we try 15 times?

Reply:
You're right. I'll make these smaller :slightly_smiling_face: